internal/pkg/es/mapping.go

```diff
@@ -1,7 +1,3 @@
-// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
-// or more contributor license agreements. Licensed under the Elastic License;
-// you may not use this file except in compliance with the Elastic License.
```

Comment:
probably will not like removed license header

Reply:
tnx